Mention grid env limitation regarding usage of Redis admin commands (#3079)
platformsh/platformsh-docs: docs/src/add-services/redis.md

redis-cli -h {{< variable "HOSTNAME" >}} -p {{< variable "PORT" >}}
```

> Please note that the admin commands `CONFIG GET` and `CONFIG SET` are restricted on the grid projects. To get the current configuration, you can use

Reviewer: ```suggestion
If you have a Grid project, note that the `CONFIG GET` and `CONFIG SET` admin commands are restricted.
To get the current configuration, run the following command:
```
